ts/de_DE.ts

```diff
@@ -317,7 +317,7 @@
     <message>
         <location filename="gui.py" line="1268"/>
         <source>In what units you want to scale?</source>
-        <translation>In welcher Einheit möchtest du skalieren?</translation>
+        <translation>In welcher Einheit möchten Sie skalieren?</translation>
```

Comment:
I still struggle with the translation of 'scene' sometimes it makes sense to use 'Motiv' sometimes 'Umgebung' sometimes 'Vorgang'. What do you think?